pyplanet/conf/default_settings.py

```diff
@@ -156,44 +171,18 @@
 
 # Dedicated contains the dedicated servers configurations, by default this is the localhost entry with default
 # credentials and details.
-DEDICATED = {
-	'default': {
-		'HOST': '127.0.0.1',
-		'PORT': '5000',
-		'USER': 'SuperAdmin',
-		'PASSWORD': 'SuperAdmin',
-	}
-}
+PYPLANET_SERVER_HOST = '127.0.0.1'
+PYPLANET_SERVER_PORT = '5000'
+PYPLANET_SERVER_USER = 'SuperAdmin'
+PYPLANET_SERVER_PASSWORD = 'SuperAdmin'
 
 # Map configuration is a set of configuration options related to match settings etc.
 # Matchsettings filename.
-MAP_MATCHSETTINGS = {
-	'default': 'maplist.txt'
-}
+PYPLANET_MATCHSETTINGS = 'maplist.txt'
 
 # Blacklist file is managed by the dedicated server and will be loaded and writen to by PyPlanet once a
 # player gets blacklisted. The default will be the filename Maniaplanet always uses and is generic.
-BLACKLIST_FILE = {
-	'default': 'blacklist.txt'
-}
-
-# The storage configuration contains the same instance mapping of the dedicated servers and is used
-# to access the filesystem on the dedicated server location.
-# Please refer to the documentation for more information.
-STORAGE = {
-	'default': {
-		'DRIVER': 'pyplanet.core.storage.drivers.local.LocalDriver',
-		'OPTIONS': {},
-	}
-}
-
-# Owners are logins of the server owners, the owners always get *ALL* the permissions in the system.
-OWNERS = {
-	'default': []
-}
-
-# Songs is a list of URLs to .ogg files of songs to be played by the music server.
-SONGS = {
-	'default': []
-}
+PYPLANET_BLACKLIST = 'blacklist.txt'
 
+# Owner, will be granted highest level of admin at initial start and connect.
+PYPLANET_OWNER = None
```

Comment:
Does this mean that there can only be one owner?
For the Smurfs we currently have two owners: Haagse and me...

Reply:
The owner is just there to grant the initial admin rights to, then this person can give the rights to the other players in-game. @TheMaximum